WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -995,4 +1005,32 @@ public void onTrimMemory(final int level) {
             }
         }
     }
+
+    private class StoryNotificationTrackerProvider implements NotificationTrackerProvider {
```

Comment:
Hey @mzorz what do you think about putting this class into another file? Just checking because this class is getting a bit big. Let me know though because it's also a really small class 🙏